config/dev/aws-credentials.yaml

```diff
@@ -19,4 +19,16 @@ type: Opaque
 stringData:
   AccessKeyID: ${AWS_ACCESS_KEY_ID}
   SecretAccessKey: ${AWS_SECRET_ACCESS_KEY}
-  SessionToken: ${AWS_SESSION_TOKEN}
\ No newline at end of file
+  SessionToken: ${AWS_SESSION_TOKEN}
+---
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: Credential
```

Comment:
Do the other providers need this too?

Reply:
Yes, but I want to get this in for AWS at least so it can make my dev cycle simpler 😄

Reply:
fwiw, they all share the same format here: `credential` in `ManagedCluster.config` and a `Credential` Kind with `identityRef` so it should be a relatively simple add.